nms/app/packages/magmalte/app/components/context/FEGSubscriberContext.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @flow strict-local
+ * @format
+ */
+
+import type {subscriber_state} from '@fbcnms/magma-api';
+
+import React from 'react';
+
+export type FEGSubscriberContextType = {
+  sessionState: {[string]: {[string]: subscriber_state}},
```

Comment:
Key type should still be `network_id`. Could you rename `sessionState` to something more meaningful? What does `session` refer to, in this case?

Reply:
I think sessionState is the appropriate name here. I am saying this because It would be good to do the same thing as what is done in the LTE network side: https://github.com/magma/magma/blob/master/nms/app/packages/magmalte/app/components/context/SubscriberContext.js#L41. This way we wouldn't create confusion for new developers if we use the same names for same thing. Also, here we are saving the subscriber state(not the subscriber him/herself) which equates to a subscriber's session. That is why session state was used in both the access network side and here. So, don't you think that keeping it as sessionState is appropriate?